runtime/vm/createramclass.cpp

```diff
@@ -1509,10 +1509,15 @@ loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9
 	UDATA packageID, BOOLEAN hotswapping, UDATA classPreloadFlags, J9Class **superclassOut, J9Module *module)
 {
 	J9JavaVM *vm = vmThread->javaVM;
+	J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
 	const BOOLEAN isROMClassUnsafe = (J9ROMCLASS_IS_UNSAFE(romClass) != 0);
 	J9UTF8 *className = J9ROMCLASS_CLASSNAME(romClass);
 	J9UTF8 *superclassName = NULL;
 	J9Class *superclass = NULL;
+	J9Class thisClass = {0};
+	thisClass.module = module;
+	thisClass.packageID = packageID;
+	thisClass.romClass = romClass;
```

Comment:
I'm not a fan of this approach.  The on-stack `thisClass`'s address will show up in tracepoint data and be confusing to anyone trying to debug this.  Also, the first thing `checkVisibility` does it try to get the current J9Class which happens to work as the obsolete flag isn't set.

I prefer adding a new helper method to visible.c that handles the lookup using just the module/packageID/romClass and having the existing method call into that one.

We've historical done this by marker the new helper as VMINLINE so the existing code can share a single implementation without incurring the overhead of a second call